canvas_sdk/handlers/simple_api/__init__.py

```diff
@@ -26,4 +27,5 @@
     "SimpleAPI",
     "SimpleAPIRoute",
     "StaffSessionAuthMixin",
+    "WebSocketAPI",
```

Comment:
ideally there is one and only one place to import a class from the SDK (this adds a second place)... we haven't been explicit about this but I think we should be... thoughts @aduane?

Reply:
So with this line, you can do either:
```python
from canvas_sdk.handlers.simple_api import WebSocketAPI
```

or

```python
from canvas_sdk.handlers.simple_api.websocket import WebSocketAPI
```

and you are advocating for just the second version?

I can see how just having a single place to import from could reduce confusion, but would we also want to remove the ability to import these other classes from here? Seems like the cat's out of the bag on this one.

Reply:
yes, just the second version is what I'd advocate (this also reduces memory usage, since the full sub-tree doesn't need to be imported unless the classes are actually used, we made this decision in home-app to both reduce circular imports and memory usage a long time ago)

while you may decide to allow both, I still think we should be explicit about it so it's consistent

Reply:
Let's be consistent with what we've done in our other codebases then and not add the import in this `__init__.py`. We won't remove the ability to import the classes already listed here, but we can at least not add to it.